ddl/table.go

```diff
@@ -76,6 +76,33 @@ func (d *ddl) onCreateTable(t *meta.Meta, job *model.Job) error {
 	}
 }
 
+func (d *ddl) delReorgTable(t *meta.Meta, job *model.Job) error {
+	// reorganization -> absent
+	tblInfo := &model.TableInfo{}
+	err := job.DecodeArgs(tblInfo)
+	if err != nil {
+		// arg error, cancel this job.
+		job.State = model.JobCancelled
+		return errors.Trace(err)
+	}
+	tblInfo.State = model.StateDeleteReorganization
+	tbl, err := d.getTable(job.SchemaID, tblInfo)
```

Comment:
Seems u have called DropTable in onDropTable function, then could u get table info here?